packages/javascript/bh-shared-ui/src/hooks/useExploreTableAutoDisplay/useExploreTableAutoDisplay.tsx

```diff
@@ -32,7 +32,7 @@ interface UseExploreTableAutoDisplayParams {
 // Auto display when current search is cypher and returned data contains nodes but not edges.
 // And dont auto display if the auto display has been closed.
 export const useExploreTableAutoDisplay = ({ enabled }: UseExploreTableAutoDisplayParams) => {
-    const { data: graphData, isFetching } = useExploreGraph();
+    const { data: graphData, isFetching } = useExploreGraph({ includeProperties: true, enabled });
```

Comment:
This appears to be the only place we are setting enabled param for the useExploreGraph hook, and im thinking it may not be needed, but I totally could be missing a use case!

If auto display table is enabled, then yes we want the useExploreGraph to be enabled. If auto display table is disabled we dont necessarily want/care to execute a graph query from this hook, however because react-query will cache the results of the graph query already, this extra call wont make an extra network request. But IMO it add a little more complexity to the useExploreGraph which Im hoping to avoid.

again its totally possible that im missing an edge case here and we need to plumb this enabled field thru!